applications/services/cli/cli_commands.c

```diff
@@ -193,12 +239,18 @@ void cli_command_log(Cli* cli, FuriString* args, void* context) {
     bool restore_log_level = false;
 
     if(furi_string_size(args) > 0) {
-        cli_command_log_level_set_from_string(args);
+        if(!cli_command_log_level_set_from_string(args)) {
```

Comment:
```
if(cli_command_log_level_set_from_string(args)){
  printf("Current log level: %s\r\n", furi_log_level_to_string(furi_log_get_level()));
} else ...
```

It will allow to remove bunch of printf's from cli_command_log_level_set_from_string, and make the code much cleaner and maintainable.

Reply:
Ah, makes sense